.circleci/owasp/workflows.yml

```diff
@@ -13,7 +13,7 @@
           filters:
             branches:
               only:
-                - develop
+                - debug/zap-resource-class-bump #switch back to develop before merge!
```

Comment:
@andrew-jameson i think this would need to be resolved first. plus we need it to run on main and master branches.